### DISCUSSION TITLE
deepin-wallpapers 许可证需要规范

### DISCUSSION THREAD

**Discussion by wineee** [linuxdeepin/developer-center#3340] [General | 常规讨论]
https://github.com/linuxdeepin/deepin-wallpapers 仓库有 3 个许可证，代码 GPLv3，deepin 目录壁纸 CC-BY-NC 3.0，deepin-community 目录壁纸 CC-BY-SA 3.0。

一些发行版打包只标注了 GPLv3
https://github.com/archlinux/svntogit-community/blob/packages/deepin-wallpapers/trunk/PKGBUILD
https://github.com/deepin-community/deepin-overlay/blob/master/dde-base/deepin-wallpapers/deepin-wallpapers-1.7.10.ebuild

需要注意的是 CC-BY-NC 是非自由的协议，在 Debian，Gentoo， NixOS 等发行版中使用会受到限制，最好把非自由协议的壁纸拆出来。

此外我发现还有 2 个壁纸仓库：
https://github.com/linuxdeepin/dde-wallpapers 标注 MIT 协议，似乎为 Debian 专门设计，但只有一张壁纸
https://github.com/linuxdeepin/deepin-wallpapers-nonfree 似乎是从 deepin-wallpapers 的 deepin 目录拆出来的，但协议居然是 CC0 1.0 Universal，这和 CC-BY-NC 3.0 是完全冲突的。

**Comment by BLumia** [comment]
有一个相关 issue 了： https://github.com/linuxdeepin/developer-center/issues/2447
